x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/ml/datafeed_high_mean_transaction_duration.json

```diff
@@ -10,5 +10,10 @@
         { "exists": { "field": "transaction.duration.us" } }
       ]
     }
-  }
+  },
+  "datafeed_config": {
+    "indices_options": {
+      "ignore_unavailable": true
+    }
+  }  
```

Comment:
@peteharverson Like this?

Reply:
You don't need the `datafeed_config` level here, it just needs to be:
```
{
  "job_id": "JOB_ID",
  "indices": [
    "INDEX_PATTERN_NAME"
  ],
  "query": {
    "bool": {
      "filter": [
        { "term": { "processor.event": "transaction" } },
        { "exists": { "field": "transaction.duration.us" } }
      ]
    }
  },
  "indices_options": {
    "ignore_unavailable": true
  }
}
```